lambdas/src/Environment.ts

```diff
@@ -124,11 +128,15 @@ export class EnvironmentBuilder {
       EnvironmentConfig.CONTENT_SERVER_ADDRESS,
       () => process.env.CONTENT_SERVER_ADDRESS
     )
-    this.registerConfigIfNotAlreadySet(
-      env,
-      EnvironmentConfig.COMMS_SERVER_ADDRESS,
-      () => process.env.COMMS_SERVER_ADDRESS
+
+    const realCommsServerAddress = await getCommsServerUrl(
+      EnvironmentBuilder.LOGGER,
+      process.env.INTERNAL_COMMS_SERVER_ADDRESS ?? DEFAULT_INTERNAL_COMMS_SERVER_URL,
+      process.env.COMMS_SERVER_ADDRESS
     )
+
+    this.registerConfigIfNotAlreadySet(env, EnvironmentConfig.COMMS_SERVER_ADDRESS, () => realCommsServerAddress)
```

Comment:
Please check where was `COMMS_SERVER_ADDRESS` being used as we are sure that we didn't break anything with this change

Reply:
This was only used  [here](https://github.com/decentraland/catalyst/blob/f2e4aa49229bc9bb6e082a1113f5609b2ccdf51c/lambdas/src/apis/status/routes.ts#L11) and since the variable is still named the same, it works